web/src/app/wizard/WizardScheduleForm.js

```diff
@@ -28,256 +29,266 @@ const styles = {
   tooltip: {
     fontSize: 12,
   },
-}
+}))
 
 /**
  * Renders the form fields to be used in the wizard that
  * can be used for creating a primary and secondary schedule.
  */
-@withWidth()
-@withStyles(styles)
-export default class WizardScheduleForm extends React.Component {
-  static propTypes = {
-    onChange: p.func.isRequired,
-    value: valuePropType,
-    secondary: p.bool,
-  }
-
-  getKey = () => {
-    const { secondary } = this.props
-    return secondary ? 'secondarySchedule' : 'primarySchedule'
-  }
-
-  handleRotationTypeChange = (e) => {
-    const { onChange, value } = this.props
-    onChange(set(value, [this.getKey(), 'rotation', 'type'], e.target.value))
-    this.forceUpdate()
-  }
-
-  handleFollowTheSunToggle = (e) => {
-    const { onChange, value } = this.props
-    onChange(
-      set(
-        value,
-        [this.getKey(), 'followTheSunRotation', 'enable'],
-        e.target.value,
-      ),
-    )
-    this.forceUpdate()
-  }
-
-  /*
-   * Choose users
-   *  - if 1 user, add as user assignment
-   *  - if 2+ users, add as rotation assignment
-   * Create shifts for assignment, if needed
-   *
-   * Ask if second assignment is needed for evening shifts
-   *   - repeat choosing users if yes
-   */
-  render() {
-    const { classes, secondary, value, width } = this.props
-    const key = secondary ? 'secondarySchedule' : 'primarySchedule'
-    const schedType = secondary ? 'secondary' : 'primary'
 
+function renderFollowTheSun(key, schedType, classes, value, width) {
```

Comment:
let's place these render functions within the component for scope, so we don't have to pass the variables around as much